engine/runtime/src/main/java/org/enso/interpreter/runtime/callable/UnresolvedConstructor.java

```diff
@@ -15,18 +18,21 @@
  */
 @ExportLibrary(InteropLibrary.class)
 public final class UnresolvedConstructor implements EnsoObject {
-  private static final Object[] NONE = new Object[0];
+  private static final CallArgumentInfo[] NONE = new CallArgumentInfo[0];
   private final String name;
+  private final CallArgumentInfo[] descs;
   private final Object[] args;
 
   /**
    * Creates a new unresolved name.
    *
    * @param name constructor name
+   * @param descs arguments to apply to it
    * @param args arguments to apply to it
```

Comment:
the doc comment for these 2 things is the same. That is suspicious.

Reply:
Done in 49202c9421